app/matplotlib.ipynb

```diff
@@ -86,23 +84,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
     "x = np.random.rand(50)\n",
     "y = np.random.rand(50)\n",
     "colors = np.random.rand(50)\n",
     "sizes = 100 * np.random.rand(50)\n",
     "\n",
-    "# write your code here"
+    "plt.scatter(x, y, c=colors, s=sizes)\n",
+    "plt.title(\"Population over year\")\n",
+    "plt.xlabel(\"Year\")\n",
+    "plt.ylabel(\"Population\")\n",
```

Comment:
The title and axis labels for the scatter plot are the same as those used in the line plot, which is incorrect. They should be specific to the scatter plot data.